services/ui-src/src/views/StateHome/__tests__/index.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { StateHome } from "../index";
+import { render, screen, fireEvent } from "@testing-library/react";
+import { RouterWrappedComp } from "utils/testing";
+
+const mockedNavigate = jest.fn();
+
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useParams: jest.fn().mockReturnValue({
+    year: "2021",
+    state: "OH",
+  }),
+  useNavigate: () => mockedNavigate,
+}));
+
+beforeEach(() => {
+  render(
+    <RouterWrappedComp>
+      <StateHome />
+    </RouterWrappedComp>
+  );
+});
+
+describe("Test StateHome", () => {
+  test("Check that the Heading renders", async () => {
+    expect(
+      await screen.findByText(/Core Set Measures Reporting/i)
```

Comment:
Could await findby be replaced with this?
```suggestion
      screen.getByText(/Core Set Measures Reporting/i)
```
This applies to the below ones as well. If not, that is fine

Reply:
yet it can, and so it shall 👍  way better